src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private String color;
+    private double side;
+
+    public Square(String color, double side) {
+        this.color = color;
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Square: color="
```

Comment:
When printing information to the console, it's best to use English for all messages to ensure the code is universal and consistent. This includes any output in the `draw()` method.